hknweb/exams/templates/exams/exams_add.html

```diff
@@ -0,0 +1,82 @@
+{% extends "base.html" %}
+{% load static %}
+{% block title %}Add an event{% endblock %}
+
+{% block header %}
+<!--<link rel="stylesheet" href="{% static "events/style.css" %}">-->
+<!--<style>-->
+    <!--.heading {-->
+        <!--display: none;-->
+    <!--}-->
+    <!--.content {-->
+        <!--padding-top: 1em;-->
+        <!--padding-bottom: 1em;-->
+    <!--}-->
+    <!--@media screen and (min-width: 1000px) {-->
+        <!--.content {-->
+            <!--padding-top: 1.5em;-->
+            <!--padding-bottom: 1.5em;-->
+        <!--}-->
+    <!--}-->
+    <!--.centered {-->
+        <!--max-width: 700px;-->
+        <!--margin: 0 auto;-->
+    <!--}-->
+<!--</style>-->
```

Comment:
Can we get rid of this? It's duplicated below.

Reply:
Removed